lib/ios/RNNComponentPresenter.m

```diff
@@ -90,6 +93,12 @@ - (void)mergeOptions:(RNNNavigationOptions *)options resolvedOptions:(RNNNavigat
 	if (options.modalPresentationStyle.hasValue) {
 		[viewController setModalPresentationStyle:[RCTConvert UIModalPresentationStyle:options.modalPresentationStyle.get]];
 	}
+
+	if (options.modalInPresentation.hasValue) {
```

Comment:
Instead of checking if `options.modalInPresentation.hasValue` - use `[withDefault.bottomTabs.visible getWithDefaultValue:NO]
Notice you should use the `withDefault` object instead of the actual `options passed to this function.

**ALSO:** please add this logic to `mergeOptions` as well 👍

Reply:
This is the code for mergeOptions, and all options use `options.modalInPresentation.hasValue`. I use `withDefault` in above. Can you look again?